intro-HPC/ch_account.tex

```diff
@@ -267,6 +283,14 @@ \subsection{Users of the \association}
 to share your Mail and Affiliation with KU Leuven (which actually means the VSC accountpage)
 Select the 'Remember' checkbox and click 'Yes, continue'.
 \fi
+
+\ifbrussel
+ \includegraphics*[width=5.73in, height=4.00in, keepaspectratio=false]{ch2-browser-authenticate-brussel}
```

Comment:
thank you for the screenshots, we should probably have a system where we can import images from their institute subdirectories? A latex macro could be created for this?
so we could put `\includeinstitutegraphics*[width=5.73in, height=4.00in, keepaspectratio=false]{ch2-browser-authenticate}` which looks into the corresponding 'institute' directory and takes the correct graphic?